akirachix/Mphavu-Informational
# Feature/featuredacademies

###  What this PR does?
This PR introduces the "Featured Academies" section to our platform. The section allows users to explore and familiarize themselves with various sports academies, providing information about each academy in a visually engaging manner using images and brief descriptions. This is designed to improve user experience by showcasing academies that users may be interested in
This PR introduces featured academies in our project. Users can use this section to famirialise themselves with the this academies. 
Perhaps answering the following questions will help create a good description:
### What is this?
* The goal of this feature is to highlight key sports academies within the platform, making it easier for users to discover and engage with these academies. It adds value by providing a clear and accessible presentation of academies users might want to interact with, driving user interest in these institutions.
 How are you doing this? 
* The feature is implemented as a new section in the frontend using the FeaturedAcademies component.
The layout is responsive, ensuring a seamless experience across various devices
- List of updates for features, enhancements, or bugfixes
New Feature: Added "Featured Academies" section to the frontend.
### Type of change

Specify the kind of PR/MR

Please delete options that are not relevant.


## How can it be tested?
Run the application and navigate to the main page where the "Featured Academies" section is implemented.
Test responsiveness by resizing the browser window or using Chrome's device toolbar to simulate various screen sizes (including Surface Pro 7 and Asus Zenbook Fold).
Ensure the image sizes adjust correctly based on screen size and that the layout remains visually appealing and functional.
Cross-browser testing: Test the layout in different browsers (Chrome, Firefox, Edge) to ensure compatibility.
Check image loading: Ensure images load quickly and render correctly.

# Where is it documented?
- List JIRA tickets
